ReactWindows/ReactNative/ReactPage.cs

```diff
@@ -140,57 +144,33 @@ protected virtual ReactRootView CreateRootView()
         }
 
         /// <summary>
-        /// Captures the key down events to 
-        /// </summary>
+        /// Captures the all key downs and Ups. 
+        /// </summary>
+        /// <param name="sender"></param>
         /// <param name="e"></param>
-        protected override void OnKeyDown(KeyRoutedEventArgs e)
+        private void OnAcceleratorKeyActivated(CoreDispatcher sender, AcceleratorKeyEventArgs e)
         {
             if (_reactInstanceManager.DevSupportManager.IsEnabled)
             {
-                if (e.Key == VirtualKey.Shift)
-                {
-                    _isShiftKeyDown = true;
-                }
-                else if (e.Key == VirtualKey.Control)
-                {
-                    _isControlKeyDown = true;   
-                }
-                else if (_isShiftKeyDown && e.Key == VirtualKey.F10)
-                {
-                    _reactInstanceManager.DevSupportManager.ShowDevOptionsDialog();
-                    e.Handled = true;
-                }
-                else if (_isControlKeyDown && e.Key == VirtualKey.R)
+                if (e.VirtualKey == VirtualKey.Shift)
                 {
-                    _reactInstanceManager.DevSupportManager.HandleReloadJavaScript();
-                    e.Handled = true;
+                    _isShiftKeyDown = e.EventType == CoreAcceleratorKeyEventType.KeyDown;
                 }
-            }
-        }
-
-        /// <summary>
-        /// Captures the key up event to potentially launch the dev options menu.
-        /// </summary>
-        /// <param name="e"></param>
-        protected override void OnKeyUp(KeyRoutedEventArgs e)
-        {
-            if (_reactInstanceManager.DevSupportManager.IsEnabled)
-            {
-                if (e.Key == VirtualKey.Menu)
+                else if (e.VirtualKey == VirtualKey.Control)
                 {
-                    _reactInstanceManager.DevSupportManager.ShowDevOptionsDialog();
-                    e.Handled = true;
+                    _isControlKeyDown = e.EventType == CoreAcceleratorKeyEventType.KeyDown;
                 }
-                else if (e.Key == VirtualKey.Shift)
+                else if (_isShiftKeyDown && (e.VirtualKey == VirtualKey.F10 || e.VirtualKey==VirtualKey.Menu))
```

Comment:
I think the intent is for menu to work without shift (that's the expected behavior on Android, at least).  I added Shift+F10 for keyboards without a Menu button.